plugins/web-terminal/src/components/TerminalComponent/TerminalComponent.tsx

```diff
@@ -128,7 +166,29 @@ export const TerminalComponent = () => {
             type="password"
             variant="outlined"
             inputRef={tokenRef}
+            required
           />
+          <Select
+            name="namespace"
+            inputRef={namespaceRef}
+            defaultValue={defaultNamespace}
+            style={{ display: isHidden ? 'none' : 'block' }}
+          >
+            {namespaces.map(namespace => {
+              return (
+                <MenuItem key={namespace} value={namespace}>
+                  {namespace}
+                </MenuItem>
+              );
+            })}
+          </Select>
+          <p
+            className={classes.centerInfo}
+            style={{ display: isHidden ? 'none' : 'block' }}
+          >
+            Unable to create devworkspace resource in {formData.namespace}{' '}
+            namespace, please choose a different one
+          </p>
```

Comment:
```suggestion
          {{ !isHidden && <>
          <Select
            name="namespace"
            inputRef={namespaceRef}
            defaultValue={defaultNamespace}
          >
            {namespaces.map(namespace => {
              return (
                <MenuItem key={namespace} value={namespace}>
                  {namespace}
                </MenuItem>
              );
            })}
          </Select>
          <p
            className={classes.centerInfo}
          >
            Unable to create devworkspace resource in {formData.namespace}{' '}
            namespace, please choose a different one
          </p>
          </>
          }}
```

Style can be easily overrided by editing HTML. If it should be hidden, please don't display that component at all.

Reply:
Resolved thanks for idea :+1: